Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandCharacterRespawn.java

```diff
@@ -29,6 +29,9 @@ public void run(Map<String, String> parameters) throws UserErrorMessage, UserReq
 		
 		CachedEntity user = db.getCurrentUser();
 		CachedEntity deadCharacter = db.getCurrentCharacter();
+		
```

Comment:
I think you want to be able to do this one, no?

Reply:
I don't think so, they get a new character with the same name when the character gets zombified. The zombie should just stay dead and do nothing.